sensor/extrinsic_calibration_manager/launch/aip_x2/ground_plane_top_unit.launch.xml

```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<launch>
+  <arg name="vehicle_id" default="default"/>
+  <let name="sensor_model" value="aip_x2"/>
+  <let name="base_frame" value="base_link"/>
+  <let name="parent_frame" value="base_link"/>
+  <let name="child_frame" value="top_unit_base_link"/>
+  <let name="pointcloud_topic" value="/sensing/lidar/left_upper/pointcloud_raw"/>
+
+  <!-- extrinsic_calibration_client -->
+  <arg name="src_yaml" default="$(find-pkg-share individual_params)/config/$(var vehicle_id)/$(var sensor_model)/sensors_calibration.yaml"/>
+  <arg name="dst_yaml" default="$(env HOME)/sensors_calibration_top.yaml"/>
+
+  <node pkg="extrinsic_calibration_client" exec="extrinsic_calibration_client" name="extrinsic_calibration_client" output="screen">
+    <param name="src_path" value="$(var src_yaml)"/>
+    <param name="dst_path" value="$(var dst_yaml)"/>
+  </node>
+
+  <!-- extrinsic_calibration_manager -->
+  <node pkg="extrinsic_calibration_manager" exec="extrinsic_calibration_manager" name="extrinsic_calibration_manager" output="screen">
+    <param name="parent_frame" value="$(var parent_frame)"/>
+    <param name="child_frames" value="
+    [$(var child_frame)]"/>
+  </node>
+
+  <!-- velodyne_rear_base_link: extrinsic_ground_plane_calibrator -->
```

Comment:
Since X2 doesn't use this, should this be `<!-- top_unit_base_link: extrinsic_ground_plane_calibrator -->` ?

Reply:
Addressed in e20b5b5fd0755825711b4f4ebb463896684f2143